src/certificates/certificate-signatories/CertificateSignatories.jsx

```diff
@@ -0,0 +1,129 @@
+import PropTypes from 'prop-types';
+import { Stack, Button, Form } from '@openedx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import CertificateSection from '../certificate-section/CertificateSection';
+import Signatory from './signatory/Signatory';
+import SignatoryForm from './signatory/SignatoryForm';
+import useEditSignatory from './hooks/useEditSignatory';
+import useCreateSignatory from './hooks/useCreateSignatory';
+import messages from './messages';
+
+const CertificateSignatories = ({
+  isForm,
+  editModes,
+  signatories,
+  arrayHelpers,
+  initialSignatoriesValues,
+  setFieldValue,
+  setEditModes,
+  handleBlur,
+  handleChange,
+}) => {
+  const intl = useIntl();
+
+  const {
+    toggleEditSignatory,
+    handleDeleteSignatory,
+    handleCancelUpdateSignatory,
+  } = useEditSignatory({
+    arrayHelpers, editModes, setEditModes, setFieldValue, initialSignatoriesValues,
+  });
+
+  const { handleAddSignatory } = useCreateSignatory({ arrayHelpers });
+
+  return (
+    <CertificateSection
+      title={intl.formatMessage(messages.signatoriesSectionTitle)}
+      className="certificate-signatories"
+    >
+      <div>
+        <p className="mb-4.5">
+          {intl.formatMessage(messages.signatoriesRecommendation)}
+        </p>
+        <Stack gap="4.5">
+          {signatories.map(({
+            name, title, organization, signatureImagePath,
+          }, idx) => (
+            isForm || editModes[idx] ? (
+              <SignatoryForm
+                // eslint-disable-next-line react/no-array-index-key
+                key={`signatory-${idx}`}
```

Comment:
Instead of disabling the lint error is there another way to make the key unique?